docs/testing-operators.md

```diff
@@ -1,120 +1,404 @@
-# Testing Operators
+# Testing your Operator with Operator Framework
 
-This document describes how operators submitted to `community-operators` are expected to be tested. Operators should be tested by their authors before submission, and will undergo [Operator Lifecycle Manager][olm] (OLM) deployment, [scorecard][sdk-scorecard], and [`operator-courier`][courier] testing in our CI environments on submission.
+These instructions walk you through how to test if your Operator deploys correctly with Operator Framework.
 
-## PR Continuous Integration
+The process below assume that you have an Kubernetes Operator in the Operator Framework *bundle* format, for example:
 
-Operators submitted to this repo are automatically tested on a Kubernetes cluster before being merged. The Kubernetes distribution used for testing depends on which directory the operator is submitted to. Ideally all tests should pass before merging.
+```
+$ ls my-operator/
```

Comment:
Should this example be a nested bundle?

Reply:
+1

Reply:
We have not moved to this bundle format yet for `community-operators`. When we do, we will update this and the rest of the instructions accordingly.